TransactionService/docker_context/apiListeners.py

```diff
@@ -20,6 +26,42 @@ async def connectRedis(app, loop):
     )
 
 
+async def initRedisHandler(app, loop):
+    app.config['redisHandler'] = RedisHandler(app.config['redisPool'], app.config['client'], config.DATABASE_SERVER_IP,

```

Comment:
would we want the redis handler to be the one that manages the connection pool instead of the service?

Reply:
I intended the redis handler to be more of a data handler, But I can combine these if you think it'd be better.